sim/src/image.rs

```diff
@@ -1374,25 +1365,37 @@ impl Images {
 }
 
 impl RamData {
+    // TODO: This is not correct. The second slot of each image should be at the same address as
+    // the primary.
     fn new(slots: &[[SlotInfo; 2]]) -> RamData {
         let mut addr = RAM_LOAD_ADDR;
         let mut places = BTreeMap::new();
+        // println!("Setup:-------------");
```

Comment:
some debug  print left in

Reply:
There are a lot of commented out debug print messages in the simulator. I propose changing these into low level log messages, but since this isn't the only place, maybe making that a separate PR.

Reply:
ACK